src/drivers/uart/src/null.rs

```diff
@@ -0,0 +1,31 @@
+// null is a simple Null driver.
+// TODO: move it out of uart.
+use core::ops;
+use model::*;
+
+pub struct Null {
```

Comment:
What about a unit-like struct?

```
pub struct Null;
```

Reply:
you know more than I, can you tell me more.